tooling/helium-server/worker/ssr.js

```diff
@@ -46,14 +46,19 @@ async function handleSsr(url, authToken = null, userAndAppearanceToken = null) {
     sha256,
     authToken
   );
-  const { httpResponse } = pageContext;
+
+  const { httpResponse, redirectTo } = pageContext;
 
   if (!httpResponse) {
     return null;
   } else {
     const { statusCode, body } = httpResponse;
     const headers = assembleHeaders(pageContext);
 
+    if (redirectTo) {
+      url = redirectTo;
+    }
+
     return new Response(resolveAssetUrls(url, body), {
```

Comment:
not sure resolveAssetUrls makes sense when we're redirecting?

Reply:
i created a default status code if one is not provided. I have also set the body in the case of the redirection